my-app/src/Components/Comments/CommentSectionContainer.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Comment from './Comment';
+import CommentInput from './CommentInput';
+
+class CommentSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: props.comments,
+      comment: ''
+    };
+  }
+
+  componentDidMount() {
```

Comment:
This is some fancy stuff! Did you come up with this yourself? Either way, this is rad